acceptance/wordpress_test.go

```diff
@@ -110,13 +110,11 @@ var _ = Describe("Wordpress", func() {
 				TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
 			},
 		}
-		resp, err := client.Do(request)
-		Expect(err).ToNot(HaveOccurred())
-		Expect(resp.StatusCode).To(Equal(http.StatusOK))
-		defer resp.Body.Close()
-
-		bodyBytes, err := ioutil.ReadAll(resp.Body)
-		Expect(err).ToNot(HaveOccurred())
-		Expect(string(bodyBytes)).To(MatchRegexp("Select a default language"))
+		Eventually(func() int {
+			resp, err := client.Do(request)
+			Expect(err).ToNot(HaveOccurred())
+			defer resp.Body.Close()
```

Comment:
The `Body` is not read/used, contrary to before the change. Why is the `Close()` still needed ?

Reply:
Removed.